common/autoware_testing/design/autoware_testing-design.md

```diff
@@ -63,3 +112,4 @@ build/<package_name>/test_results/<package_name>/<executable_name>_smoke_test.xu
 # Related issues
 - Issue #700: add smoke test
 - Issue #1224: Port other packages with smoke tests to use `autoware_testing`
+- Issue #951: Add infrastructure for component-interface test
```

Comment:
It's better to replace these links with the links of Autoware.Auto because it's now on GitHub.

Reply:
Added Autoware.Auto's link 7faa04c1bacbdca48882c727df5c0ac56733d38a